internal/controllers/utils/utils.go

```diff
@@ -602,6 +601,20 @@ func ExtractTemplateDataFromConfigMap[T any](ctx context.Context, c client.Clien
 	return validData, nil
 }
 
+// ExtractTimeoutConfigFromConfigMap extracts the timeout config from the ConfigMap by key if exits,
+// converting it from integer to time.Duration. Returns an error if the value is not a valid integer.
+func ExtractTimeoutConfigFromConfigMap(cm *corev1.ConfigMap, key string) (time.Duration, error) {
```

Comment:
I think this function can be removed and instead use GetConfigMapField (https://github.com/openshift-kni/oran-o2ims/pull/238/files#diff-ff9c80a7f6ade29a53eec0fe48e61578a5c73fac596e79b467b386fa3cc8873bR629) from Allain's PR or just keep this function, but call `GetConfigMapField` and convert to int.

It should still be renamed to something more similar, like `GetConfigMapTimeout` / `GetTimeoutFromConfigMap`

Reply:
Will rename and update to use GetConfigMapField